cyclus/system.py

```diff
@@ -0,0 +1,56 @@
+"""Some system-specific info for cyclus."""
+import sys
+import importlib
+
+from cyclus.lazyasd import lazyobject
+
+
+@lazyobject
+def PY_VERSION_TUPLE():
+    return sys.version_info[:3]
+
+
+@lazyobject
+def curio():
+    if PY_VERSION_TUPLE < (3, 5, 2):
+        return None
+    else:
+        try:
+            return importlib.import_module('curio')
+        except ImportError:
+            return None
```

Comment:
So I did it this way because the `loop()` code is called every tine step of all cyclus simulations and these imports do not support all the same versions of Python that Cyclus supports.  In normal cases it shouldn't be an issue if these exist or not, which is why None is preferable over an error.  

In the case that it does matter, when starting up cyclus.server, if these packages don't exist there will be an error because SimState cannot be instantiated.  

So this works when it is supposed to and fails when it should.